documentation/messages/user.registration.transactional.md

```diff
@@ -0,0 +1,123 @@
+# User registration transactional message
+
+User registration event.
+
+**Routing key**
+
+`user.registration.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Queues and binding keys**
+
+| Queue                      | Binding key           |
+| -------------------------- | --------------------- |
+| `activityStatsQueue`       | `*.*.transactional`   |
+| `loggingQueue`             | `*.*.transactional`   |
+| `mobileCommonsQueue`       | `campaign.signup.*`   |
```

Comment:
`campaign.signup.*` binding does not apply here. Change to second key binding `user.registration.*`